src/cli-validator/utils/unsupportedOptionError.js

```diff
@@ -0,0 +1,23 @@
+var program = require('commander');
```

Comment:
You should never use `var` in modern JS. Use `const` for everything that won't change and `let` for any variable you will modify

```suggestion
const program = require('commander');
```